templates/base.html

```diff
@@ -69,5 +69,7 @@
 </script>
 {% endblock %}
 <script src="{% static 'js/script.js' %}"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js" integrity="sha512-uto9mlQzrs59VwILcLiRYeLKPPbS/bT71da/OEBYEwcdNUk8jYIy+D176RYoop1Da+f9mvkYrmj5MCLZWEtQuA==" crossorigin="anonymous">

```

Comment:
Could you check if this is necessary or if you can remove `$( "#accordion" ).accordion();` from script.js. I don't think that is actually required for the accordion to work.

Reply:
True, I'll delete the last script in script.js and the import as the accordion will work without them both.